src/mail.sh

```diff
@@ -761,8 +787,23 @@ function parse_mail_options()
         options_values['NO_INTERACTIVE']=1
         shift
         ;;
+      -[[:digit:]])
+        # if this is given without a commit reference we need to avoid adding a
+        # assumed commit
+        commit_count="$1"
+        commit_ref=1
+        shift
+        ;;
+      -v)
+        patch_version="$1$2"
```

Comment:
`"$1$2"`? Could you elaborate a little bit about it? Also, add a comment about that in this part of the code.

Reply:
After going through `kw_parse` the argument `-vX` will be split in `-v` and `X`, here I am simply joining them again to pass over to `git send-email`.